features/persistence.feature

```diff
@@ -127,3 +127,11 @@ Feature: Trace and state persistence
     # And I wait to receive a sampling request
     # And I discard the oldest sampling request
     # And I wait to receive a sampling request
+
+  @skip_webgl #Pending PLAT-8151
+  Scenario: P value response parsing
```

Comment:
Is it worth having a test the specifically tries a different locale?  Or perhaps even all the tests should run in multiple locales?

Reply:
@twometresteve is that possible?